src/main/kotlin/com/openlattice/shuttle/pods/ShuttleServicesPod.kt

```diff
@@ -123,6 +124,9 @@ class ShuttleServicesPod {
     @Inject
     private lateinit var byteBlobDataManager: ByteBlobDataManager
 
+    @Inject
+    private lateinit var externalDbConnMan: ExternalDatabaseConnectionManager
```

Comment:
Does this need `ExternalDatabaseConnectionManagerPod` added in order to inject this? Or does that get called through something else?

Reply:
Yeah, I'll add it to ShuttleServer. Haven't run into issues b/c I haven't been running ShuttleServer